core/src/main/resources/reference.conf

```
@@ -544,7 +544,7 @@ jdbc-read-journal {
 
 # the akka-persistence-durable-state-store in use
 jdbc-durable-state-store {
-  class = "akka.persistence.jdbc.state.scaladsl.DurableStateStore"
+  class = "akka.persistence.jdbc.state.scaladsl.JdbcDurableStateStore"
```

I think the correct value for this setting is

```suggestion
  class = "akka.persistence.jdbc.state.JdbcDurableStateStoreProvider"
```